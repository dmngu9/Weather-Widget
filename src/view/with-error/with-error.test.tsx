import * as React from 'react';
import { shallow } from 'enzyme';

import withError from './with-error';

const Example: React.StatelessComponent<{ title: string }> = ({ title }) => <div>{title}</div>;

const WithError = withError(Example);

describe('WithError', () => {
    it('should render correctly with error', () => {
        const wrapper = shallow(<WithError error="this is error" title="title" />);

        expect(wrapper).toMatchSnapshot();
    });

    it('should render correctly when no error', () => {
        const wrapper = shallow(<WithError title="title" />);

        expect(wrapper).toMatchSnapshot();
    });
});
