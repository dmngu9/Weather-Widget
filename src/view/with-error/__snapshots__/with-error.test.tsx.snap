// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithError should render correctly when no error 1`] = `
<Example
  title="title"
/>
`;

exports[`WithError should render correctly with error 1`] = `
<styled.p>
  this is error
</styled.p>
`;
